tools/rewriter/GenCallAsm.cpp

```diff
@@ -657,13 +707,13 @@ std::string emit_asm_wrapper(const CAbiSignature &sig,
     start_of_ret_space += 8;
 
     // See what unpadded alignment would be for the start of the stack return.
-    size_t unaligned = start_of_ret_space % stack_return_align;
+    size_t unaligned = (start_of_ret_space + stack_return_size) % stack_return_align;
```

Comment:
This comment doesn't seem to make sense anymore: the start of the stack return wouldn't include its size.

Reply:
Good point, this make more sense?